pkg/project/create.go

```diff
@@ -323,9 +329,20 @@ func writeNewCwSettings(conID string, pathToCwSettings string, BuildType string)
 	return nil
 }
 
-func getDefaultCwSettings(conID string, BuildType string) CWSettings {
+func getDefaultCwSettings(conID string, BuildType string) (CWSettings, error) {
```

Comment:
Should this not return a `ProjectError`?

Reply:
There was a bit of a mix through the file, I've gone through and wherever we catch an external error (i.e from io, our own connections package), changed it to immediately create the projectError.